src/installation/index.md

```diff
@@ -1,117 +1,129 @@
 # Setting Up a Development Environment
 
-With an understanding of the ecosystem surrounding Rust on Espressif chips, we can move on to actual development. If you are not aware of the two possible development approaches or do not understand the differences between writing `std` and `no_std` applications, please first read the [Ecosystem Overview] chapter.
+At the moment, Espressif SoCs are based on two different architectures -- `RISC-V` and `Xtensa`. Both architectures support `std` and `no_std` approaches.
 
-Let's take a moment to discuss the Rust support for the different architectures of the Espressif chips and the different approaches in more detail. At this moment, Espressif SoCs are based on two different architectures: `RISC-V` and `Xtensa`. The support for those two architectures in the Rust programming language is very different.
+To set up the development environment, do the following:
 
-[Ecosystem Overview]: ../overview/index.md
+1. [Install Rust][install-rust]
+2. Install requirements based on your target(s)
```

Comment:
@SergioGasquez 

Do you think we can update the list like this (as well as the respective section names below):

- `RISC-V` targets only
- `Xtensa` and `RISC-V` targets

## RISC-V targets only

## Xtensa and RISC-V targets